tutorial/tutorial08.yml

```diff
@@ -35,6 +50,8 @@ cables:
     colors: [YE, BK, BK, RD]
     manufacturer: [WiresCo,WiresCo,WiresCo,WiresCo] # set a manufacter per wire
     mpn: [W1-YE,W1-BK,W1-BK,W1-RD]
+    supplier: [WireShack,WireShack,WireShack,WireShack]
```

Comment:
When all wires in a bundle have the same attribute value, it's also possible to specify this attribute value as a single string. In that case, the common value will be shown in the top part of the node, though. The BOM should be the same in both cases.

Reply:
Should we offer an option to always show such a common attribute value for each wire in a bundle?

Reply:
I think the current way is fine.
If the attribute is a string, display it once in the same row as `length` etc.
If it is a list, display one list item per wire.